.golangci.yml

```
@@ -1,24 +1,23 @@
 linters:
   # Full list available at https://golangci-lint.run/usage/linters/#disabled-by-default
+  disable-all: true
```

@okdas Why keep the linters here if it's disabled?

1. We should delete it
OR
2. Add the comment below

Wanted to understand why not do (1)

```suggestion
  disable-all: true # Disabled and kept around as a reference. See # golangci-lint.yml instead
```

---

@Olshansk, it doesn't matter if we keep it or remove it. We list the linters we enable, even the ones enabled by default. I introduced it to try to remove the `typecheck` linter, but it seems it's not possible without CLI voodoo. Let's remove it if it causes any confusion.

---

> Let's remove it if it causes any confusion.

Correct. Dead code -> raises questions -> increases cognitive load -> confuses developers -> leads to time spent on irrelevant things -> reduces productivity